app/helpers/reactions_helper.rb

```diff
@@ -0,0 +1,14 @@
+module ReactionsHelper
+  def reaction_like_icon
+    icon 'thumbs-up fa-fw', 'Like'
+  end
+
+  def reaction_love_icon
+    icon 'heart fa-fw', 'Love'
+  end
+
+  def reaction_genius_icon
+    icon 'lightbulb-on fa-fw', 'Genius'
+  end
```

Comment:
Is Genius meant to be the superlative in like - love -genius? 
Also, the connotation with Genius is 'magic', and with lightbulb is : 'new idea' (Oprah's light bulb moment :joy)

Reply:
We're actually getting rid of it altogether now (decided today).